app/views/admin/configs/show.html.erb

```diff
@@ -1191,61 +1191,61 @@
+          <%= admin_config_label :home_feed_minimum_score, model: Settings::Experience %>
+          <%= admin_config_description Constants::Settings::UserExperience::DETAILS[:home_feed_minimum_score][:description] %>
           <%= f.number_field :home_feed_minimum_score,
                              class: "crayons-textfield",
-                             value: SiteConfig.home_feed_minimum_score,
-                             placeholder: Constants::SiteConfig::DETAILS[:home_feed_minimum_score][:placeholder] %>
+                             value: Settings::UserExperience.home_feed_minimum_score,
+                             placeholder: Constants::Settings::UserExperience::DETAILS[:home_feed_minimum_score][:placeholder] %>
       </div>
       <div class="crayons-field">
-        <%= admin_config_label :default_font %>
-        <%= admin_config_description Constants::SiteConfig::DETAILS[:default_font][:description] %>
+        <%= admin_config_label :default_font, model: Settings::Experience %>
+        <%= admin_config_description Constants::Settings::UserExperience::DETAILS[:default_font][:description] %>
         <%= select_tag "site_config[default_font]",
                        options_for_select(
                          [%w[sans-serif sans_serif], %w[serif serif], %w[open-dyslexic open_dyslexic]],
-                         SiteConfig.default_font,
+                         Settings::UserExperience.default_font,
                        ),
                        multiple: false,
                        class: "selectpicker" %>
     </div>
     <div class="crayons-field">
-      <%= admin_config_label :primary_brand_color_hex %>
-      <%= admin_config_description Constants::SiteConfig::DETAILS[:primary_brand_color_hex][:description] %>
+      <%= admin_config_label :primary_brand_color_hex, model: Settings::Experience %>
+      <%= admin_config_description Constants::Settings::UserExperience::DETAILS[:primary_brand_color_hex][:description] %>
       <%= f.color_field :primary_brand_color_hex,
                         class: "crayons-color-selector",
-                        value: SiteConfig.primary_brand_color_hex,
+                        value: Settings::UserExperience.primary_brand_color_hex,
                         pattern: "^#+([a-fA-F0-9]{6})$",
-                        placeholder: Constants::SiteConfig::DETAILS[:primary_brand_color_hex][:placeholder] %>
+                        placeholder: Constants::Settings::UserExperience::DETAILS[:primary_brand_color_hex][:placeholder] %>
   </div>
   <div class="crayons-field crayons-field--checkbox">
-    <%= f.check_box :public, checked: SiteConfig.public, class: "crayons-checkbox" %>
+    <%= f.check_box :public, checked: Settings::UserExperience.public, class: "crayons-checkbox" %>
     <div class="mt-0">
-      <%= admin_config_label :public %>
+      <%= admin_config_label :public, model: Settings::Experience %>
```

Comment:
And finally, here! 😄 :
```suggestion
      <%= admin_config_label :public, model: Settings::UserExperience %>
```

Reply:
Just finished testing this field out locally and while changing the model name here to `Settings::UserExperience` fixes the `NameError`, it does not resolve the `RoutingError` (`uninitialized constant Admin::Settings::UserExperiencesController
Did you mean?  Admin::UsersController`) that occurs when you attempt to update any of the fields within "User Experience and Brand"... 👀

Reply:
Again, this wasn't finished, the controller really doesn't exist yet but I already know the name/path because it will follow the same pattern as the previous 5 PRs 😃